README.rst

```diff
@@ -45,11 +56,39 @@ upon by some of the world's leading scientists and engineers. If you need to
 manipulate numbers on a computer and display or publish the results, give
 SciPy a try!
 
-For the installation instructions, see INSTALL.rst.txt_.
+For the installation instructions, see `our install
+guide <https://scipy.github.io/devdocs/getting_started.html#installation>`__.
+
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
```

Comment:
We are not calling like a conference anyways so this is a bit academic. People who have none of these are also welcome. Probably we need a more neutral language here. I think the old sentence can be used here.

Reply:
Thanks for having a look! I took all of this from NumPy, so sure we can adapt it as we want. I just like to be consistant between the 2. What would you propose here?

Reply:
The old sentence (removed now) is pretty OKish and neutral don't you think?